<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filter</class>
 <widget class="QMainWindow" name="Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1910</width>
    <height>1181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>常用的几种滤波</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKWidget" name="qvtkWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>391</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget_3" native="true">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>40</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget_4" native="true">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>40</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget_5" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>391</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget_6" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>570</y>
      <width>391</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget_7" native="true">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>600</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKWidget" name="qvtkWidget_8" native="true">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>600</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PassThrough</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>VoxelGrid</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>0</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>StatisticalOutlierRemoval </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ModelCoefficients </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>0</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ExtractIndices</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>550</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ConditionalRemoval </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>550</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RadiusOutlierRemoval</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1500</x>
      <y>560</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CropHull</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>390</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>统计分析</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>StddevMulThresh</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SMTBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="SODoBUT">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>执行过滤</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SOLoadBUT">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>载入点云</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MeanK</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="MKBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="value">
      <double>50.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>体素化栅格下采样</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>LeafSize</string>
     </property>
    </widget>
    <widget class="QPushButton" name="VGLoadBUT">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>载入点云</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="leafSizeSBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="VGDoBUT">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>执行过滤</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>400</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>基于随机一致性分割算法</string>
    </property>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>DistanceThreshold</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="DTBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MaxIterations</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="MIBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
     <property name="value">
      <double>1000.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="MCLoadBUT">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>载入点云</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MCDoBUT">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>执行过滤</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>直通滤波</string>
    </property>
    <widget class="QPushButton" name="PTLoadBUT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>载入点云</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PTDoBUT">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>执行过滤</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="ULBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>8.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Upper Limit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lower Limit</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="LLBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>过滤范围内</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>930</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>自定义条件滤除</string>
    </property>
    <widget class="QPushButton" name="CRLoadBUT">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>载入点云</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CRDoBUT">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>执行过滤</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z方向 GT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z方向 LT</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="LTBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.800000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="GTBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>930</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>密度滤除</string>
    </property>
    <widget class="QPushButton" name="RORLoadBUT">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>载入点云</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RORDoBUT">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>执行过滤</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="LJBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>邻近点集数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>半径范围</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="BJBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.900000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>930</y>
      <width>381</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>投影模型</string>
    </property>
    <widget class="QPushButton" name="EIDoBUT">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>执行过滤</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EILoadBUT">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>载入点云</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>940</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>多边形内部点云提取</string>
    </property>
    <widget class="QPushButton" name="CHDoBUT">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>执行过滤</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CHLoadBUT">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>载入点云</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="aBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>函数维度</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="bBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>凸包维度</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1910</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
